tools/justfile

```diff
@@ -0,0 +1,144 @@
+# Enable positional args
+set positional-arguments
+
+# Default values for the deployment
+tag    := "test"
+queue  := "active"
+worker := "htcmock"
+
+# Base compose file
+export COMPOSE_BASE := "../docker-compose/docker-compose.yml"
+
+# loggin level of deployment
+export LOGGING_LEVEL := "Information"
+
+# Sets the queue
+export QUEUE := if queue == "rabbit" {
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if queue == "rabbit091"{
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if queue == "artemis"{
+  "../docker-compose/docker-compose.queue-artemis.yml"
+}else{
+  "../docker-compose/docker-compose.queue-activemqp.yml"
+}
+
+
+# Sets the override to feed docker-compose
+export OVERRIDE := if queue == "rabbit091" {
+  "../docker-compose/docker-compose.override-rabbitmq091.yml"
+}else{
+  "../docker-compose/docker-compose.override.yml"
+}
+
+# Defines worker and enviroment variables for deployment
+export ARMONIK_WORKER := if worker == "stream" {
```

Comment:
What if I have another one that is not there ?

Reply:
That case is not handled. A possible solution could be to load a `.env` file specifying the two variables `ARMONIK_WORKER` and `WORKER_DOCKER_FILE`, is such file is not provided we set them in the script.

Reply:
Isn't it possible to do so with command parameters ?

Reply:
I'll re-check the doc and see

Reply:
I think that you can change the value of the variables directly in cli if you want.

Reply:
just pushed a solution with the env vars

Reply:
That nice like this.